done with realisation for file data source class.

anryts opened the pull request:
additionally added an infinity loop for reading from file(in _read_accelerometer_data and _read_gps_data functions).

anryts committed:
done with realisation.

additionally added a infinity loop for reading from file.

mkhusid commented:
@anryts Looks good! From a quick review - please consider changing "startReading" & "stopReading" names to match snake_case. I will also try to run it on my machine and see if I can add any improvements from my side.

anryts committed:
Renamed name of methods: start_reading() and finish_reading().
Also, moved to English for comments in code.
Updated gitignore for mosquitto runtime files (db and log)

mkhusid committed:
code improvements
